packages/code/src/Code.tsx

```diff
@@ -229,7 +230,7 @@ function Code({
   const [copied, setCopied] = useState(false);
   const [showCopyBar, setShowCopyBar] = useState(false);
   const mode = darkMode ? Mode.Dark : Mode.Light;
-  const isMultiline = useMemo(() => children.includes('\n'), [children]);
+  const isMultiline = useMemo(() => children.trim().includes('\n'), [children]);
```

Comment:
Should we add a test to prove that this works as expected and ensure that we don't introduce any regressions here?

Reply:
I can add something small to test this. To be honest, there's not a lot to test here though.

Reply:
I think this `Fixes issue where code examples would be detected as multi-line when a single line example has preceding or subsequent new line characters.` could be testable

Reply:
I can test that this bit of code checks for that effectively, but testing that the end result is correct is a bit tricky since the only change is CSS.

Reply:
can you check the number of lines rendered when that character is appended/prepended

Reply:
This code doesn't change the lines rendered at all.

Reply:
I thought that was what was called out in the changset

Reply:
To clarify, this boolean only changes CSS – there's no change to the DOM to test.

Reply:
No – we just weren't applying single line CSS styles when the passed code example had preceding, or subsequent line breaks despite us stripping those lines in the plugin. To be clear, that bit is already tested I believe.

Reply:
Got it, think I misunderstood